dogs.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles/main-style.css">
+    <link rel="stylesheet" media="screen and (max-width: 500px)" href="styles/width-small.css">
+    <link rel="stylesheet" media="screen and (min-width: 501px) and (max-width: 850px)" href="styles/width-medium.css">
+    <link rel="stylesheet" media="screen and (min-width: 851px)" href="styles/width-large.css">
+    <title>Our Dogs</title>
+</head>
+<body>
+        <header class="header-container">
+                <div class="header-item-1"><img src="images/cart-icon.svg" alt="cart icon">
+                <input class="header-item-2" type="button" value="Checkout"></div>
+                <div class="header-item-3"><img src="images/adoptadog-logo.svg" alt="dog logo"></div>
+                <h3 class="header-item-4 home-h">Seattle's priemer dog adoption agency</h3>
+            </header>
+    <nav>
+        <ul class="nav-ul">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="dogs.html">Dogs</a></li>
+            <li><a href="blog.html">Dog Blog</a></li>
+            <li><a href="contact.html">Contact</a></li>
+        </ul>
+    </nav>
+    <section>
+        <h1>Here are all of our Dogs!</h1>
+        <div class="card-parent">
```

Comment:
maybe you can find a more semantic name for this?

This organization and layout is smart, it'll make your layout work great!